src/main/java/com/redhat/devtools/lsp4ij/operations/codelens/LSPCodeLensProvider.java

```diff
@@ -0,0 +1,181 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Red Hat, Inc.
```

Comment:
2024

Reply:
This class (which has been renamed) has been created in 2021